web/pdf_page_view.js

```diff
@@ -413,10 +413,11 @@ class PDFPageView {
     const tree = await (!this.structTreeLayer.renderingDone
       ? this.pdfPage.getStructTree()
       : null);
-    const treeDom = this.structTreeLayer?.render(tree);
+    const treeDom = this.structTreeLayer.render(tree);
     if (treeDom) {
       this.canvas?.append(treeDom);
     }
+    this.structTreeLayer.show();
```

Comment:
Given the comment above:
```suggestion
    this.structTreeLayer?.show();
```